packages/btc-wallet/utils/psbt.ts

```diff
@@ -0,0 +1,67 @@
+import * as bitcoin from 'bitcoinjs-lib'
+import coinSelect from 'coinselect'
+import { esploraClient } from 'esplora-client'
+
+import { type Unisat } from '../unisat'
+
+/**
+ * Manually construct, sign and push the transaction.
+ */
+// eslint-disable-next-line complexity
+export async function sendBitcoin(
+  provider: Unisat,
+  toAddress: string,
+  satoshis: number,
+  options: { memo?: string } = {},
+) {
+  const [address] = await provider.getAccounts()
+  const network = await provider.getNetwork()
```

Comment:
It seems we can run these 2 in parallel

```suggestion
  const [[address], network] = await Promise.all([
    provider.getAccounts(),
    provider.getNetwork()
  ])
 
```